src/Eleventy.js

```diff
@@ -548,6 +548,14 @@ Arguments:
     benchmark.before();
     await this._initWatchDependencies();
     benchmark.after();
+
+    // Error overlay in development
+    this.config.events.on("beforeWatch", () => {
+      this.eleventyServe.clearError();
+    });
+    EleventyErrorHandler.events.on("errorLogged", error => {
```

Comment:
Why not just name the error `error`? It turns out that `EventEmitter` will automatically throw errors attached to events named `error`. Lots of confusion resulted from that behavior :)